### ISSUE TITLE
Error on Install 1.0@rc

### ISSUE THREAD

**Issue by raphaeldealmeida** [zendframework/zend-expressive-skeleton#46]
When I try to install the following error is returned:
```
Remove installer
Removing Expressive installer classes and configuration
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package zendframework/zend-expressive could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by weierophinney** [comment]
Can you copy the command you used to install?

**Comment by raphaeldealmeida** [comment]
composer create-project zendframework/zend-expressive-skeleton:^1.0@rc expressive

And all default options

**Comment by xtreamwayz** [comment]
@raphaeldealmeida Can you paste the complete output from the installer? So basically everything after the command until the error.

**Comment by raphaeldealmeida** [comment]
```
raphaelrsa@prateado:~/projects$ composer create-project zendframework/zend-expressive-skeleton:^1.0@rc expressive
Installing zendframework/zend-expressive-skeleton (1.0.0rc4)
  - Installing zendframework/zend-expressive-skeleton (1.0.0rc4)
    Downloading: 100%         

Created project in expressive
Setup data and cache dir
Setting up optional packages

  Minimal skeleton? (no default middleware, templates, or assets; configuration only)
  [y] Yes (minimal)
  [n] No (full; recommended)
  Make your selection (No): 

  Which router you want to use?
  [1] Aura.Router
  [2] FastRoute
  [3] Zend Router
  Make your selection or type a composer package name and version (FastRoute): 
  - Adding package zendframework/zend-expressive-fastroute (^1.0)
  - Copying /config/autoload/routes.global.php

  Which container you want to use for dependency injection?
  [1] Aura.Di
  [2] Pimple-interop
  [3] Zend ServiceManager
  Make your selection or type a composer package name and version (Zend ServiceManager): 
  - Adding package zendframework/zend-servicemanager (^2.5)
  - Adding package ocramius/proxy-manager (^1.0)
  - Copying /config/container.php

  Which template engine you want to use?
  [1] Plates
  [2] Twig
  [3] Zend View installs Zend ServiceManager
  [n] None of the above
  Make your selection or type a composer package name and version (n): 

  Which error handler do you want to use during development?
  [1] Whoops
  [n] None of the above
  Make your selection or type a composer package name and version (Whoops): 
  - Adding package filp/whoops (^1.1)
  - Copying /config/autoload/errorhandler.local.php
Remove installer
Removing Expressive installer classes and configuration
Loading composer repositories with package information
Installing dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package zendframework/zend-expressive could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
raphaelrsa@prateado:~/projects$ 
```

**Comment by xtreamwayz** [comment]
@raphaeldealmeida That looks alright. Can you paste the composer.json file as well please? The version right after you get the error.

**Comment by raphaeldealmeida** [comment]
Its songs like composer bug :/
```
{
    "name": "zendframework/zend-expressive-skeleton",
    "type": "project",
    "homepage": "https://github.com/zendframework/zend-expressive-skeleton",
    "license": "BSD-3-CLAUSE",
    "authors": [
        {
            "name": "Geert Eltink",
            "homepage": "https://xtreamwayz.github.io/"
        }
    ],
    "extra": {
        "branch-alias": {
            "dev-master": "1.0-dev",
            "dev-develop": "1.1-dev"
        }
    },
    "require": {
        "php": "^5.5 || ^7.0",
        "roave/security-advisories": "dev-master",
        "zendframework/zend-expressive": "~1.0.0@rc || ^1.0",
        "zendframework/zend-expressive-helpers": "^1.2",
        "zendframework/zend-stdlib": "~2.7",
        "zendframework/zend-expressive-fastroute": "^1.0",
        "zendframework/zend-servicemanager": "^2.5",
        "ocramius/proxy-manager": "^1.0"
    },
    "require-dev": {
        "phpunit/phpunit": "^4.8",
        "squizlabs/php_codesniffer": "^2.3",
        "filp/whoops": "^1.1"
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/App/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "AppTest\\": "test/AppTest/"
        }
    },
    "scripts": {
        "check": [
            "@cs",
            "@test"
        ],
        "cs": "phpcs",
        "cs-fix": "phpcbf",
        "serve": "php -S 0.0.0.0:8080 -t public/",
        "test": "phpunit"
    }
}
```

**Comment by xtreamwayz** [comment]
That file looks alright too. If I paste it into composer.json in an empty dir and run ``composer update``, it runs fine.

My guess is that it is an issue with composer. I have no idea how to fix it. What you can try is clearing the composer cache: https://getcomposer.org/doc/03-cli.md#clear-cache

**Comment by raphaeldealmeida** [comment]
Fix cleaning the cache failed. I got the same message.

I install zend-expressive-skeleton (0.5.3) - Stable and run perfectly. This bug just occurs on RC.

**Comment by weierophinney** [comment]
@raphaeldealmeida That's really, really strange; I cannot reproduce it either here or on Travis. :frowning: 

We should have a stable version by next week; that should solve the issues, though it would be great to get you able to test now.

The only other thing I can think of: have you done a `composer self-update` recently? I know that there was a fix after we did RC1 that changed how composer looked for stable packages, and that might fix the situation for you. Let me know the results!

**Comment by kingjerod** [comment]
@weierophinney Looks like a composer update fixes it. I had the same error as @raphaeldealmeida.

**Comment by RalfEggert** [comment]
Can this be closed since 1.0 release of `Zend\Expressive` was released?

**Comment by xtreamwayz** [comment]
Closing. If you still have problems with the stable release, please create a new issue.
